scss/layout/_grid.scss

```diff
@@ -68,6 +69,16 @@
   }
 }
 
+.slab--primary .grid {
+  a {
```

Comment:
@jenniferthibault: the bottom of the press page uses cards in a grid in a slab.
<img width="738" alt="screen shot 2016-12-20 at 3 51 31 pm" src="https://cloud.githubusercontent.com/assets/24054/21371040/3e302302-c6cc-11e6-8a59-73ca32a7c597.png">

I'm in favor of merging this in and then making a special border class for grids instead of automatically having grids bordered in slabs in a later branch (with the other homepage design tweaks) to get all the homepage grids updated.